lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
True if in the right folder it makes sense. Just wanted it to be very clearly different than nonlinearity.Then I'm agreed.